lib/core/unittest/CPackedBitVectorTest.cc

```diff
@@ -42,6 +44,200 @@ std::string toBitString(const T& v) {
     }
     return result.str();
 }
+
+class CPackedBitVectorInternals : public core::CPackedBitVector {
+public:
+    static std::size_t maximumOneByteRunLength() {
+        return MAXIMUM_ONE_BYTE_RUN_LENGTH;
+    }
+    static std::size_t maximumTwoByteRunLength() {
+        return MAXIMUM_TWO_BYTE_RUN_LENGTH;
+    }
+    static std::size_t maximumThreeByteRunLength() {
+        return MAXIMUM_THREE_BYTE_RUN_LENGTH;
+    }
+    static std::size_t maximumFourByteRunLength() {
+        return MAXIMUM_FOUR_BYTE_RUN_LENGTH;
+    }
+    static void appendRun(std::size_t runLength, std::uint8_t& lastRunBytes, TUInt8Vec& runLengthBytes) {
+        core::CPackedBitVector::appendRun(runLength, lastRunBytes, runLengthBytes);
+    }
+    static void extendLastRun(std::size_t runLength, std::uint8_t& runBytes, TUInt8Vec& runLengthBytes) {
+        core::CPackedBitVector::extendLastRun(runLength, runBytes, runLengthBytes);
+    }
+    static std::uint8_t bytes(std::size_t runLength) {
+        return core::CPackedBitVector::bytes(runLength);
+    }
+    static std::size_t readLastRunLength(std::uint8_t lastRunBytes,
+                                         const TUInt8Vec& runLengthBytes) {
+        return core::CPackedBitVector::readLastRunLength(lastRunBytes, runLengthBytes);
+    }
+    static std::size_t readRunLength(TUInt8VecCItr runLengthBytes) {
+        return core::CPackedBitVector::readRunLength(runLengthBytes);
+    }
+    static std::size_t popRunLength(TUInt8VecCItr& runLengthBytes) {
+        return core::CPackedBitVector::popRunLength(runLengthBytes);
+    }
+    static void writeRunLength(std::size_t runLength, TUInt8VecItr runLengthBytes) {
+        core::CPackedBitVector::writeRunLength(runLength, runLengthBytes);
```

Comment:
Is there a reason for redefining the methods? It doesn't look like you are mocking anything particular.

Reply:
Just to make them public in the tests: they're protected.